src/store/actionCreators/profileData.ts

```diff
@@ -135,7 +152,10 @@ export const profilePatientAddAppointment = (idDate: string) => {
           data: idDate,
         },
         {
-          headers: { Authorization: `Bearer ${cookies.getCookie('token')}` },
+          headers: {
+            Authorization: `Bearer ${cookies.getCookie('token')}`,
```

Comment:
should be stored in httpOnly cookies and taken on the server from there, do not set them manually on the client side